src/book/03-guides/04-general/04-visualmesh.mdx

```diff
@@ -114,7 +149,7 @@ mkdir output
 cp example_net.yaml ./output/config.yaml
 ```
 
-With that, we're almost ready to train the network. To save time, change the configuration to run less training epochs (500 epochs can easily take 30 hours to complete on CPU). Open the config file in a text editor, (e.g. `nano output/config.yaml`) and change the line that says `epochs: 500` to say `epochs: 5`. Also note that the paths for the training, testing and validation tfrecords are set in the config file.
+With that, we're almost ready to train the network. To save time, change the configuration to run less training epochs (500 epochs can easily take 30 hours to complete on CPU). Open the config file in a text editor, (e.g. `nano output/config.yaml`) and change the line that says `epochs: 500` to say `epochs: 5`. Also note that the paths for the training, testing and validation tfrecords are set in the config file. *If you have accidentally started training 500 epochs hit `ctrl+c` to keyboard interrupt training, if that doesn't work, open another bash terminal, type `xkill` then click on the training terminal window*
```

Comment:
As you can likely guess, I accidentally didn't save the file in nano and wondered why the estimate was in days, just figured that if a beginner is doing this they may not know how to easily stop the training